go/storage/mkvs/syncer/proof.go

```diff
@@ -113,7 +116,10 @@ func (b *ProofBuilder) Size() uint64 {
 
 // Build tries to build the proof.
 func (b *ProofBuilder) Build(ctx context.Context) (*Proof, error) {
-	var proof Proof
+	proof := Proof{
+		Versioned: cbor.NewVersioned(1),
```

Comment:
Add a `LatestProofVersion` constant.

Also probably we would need the builder to support both V0 and V1, so that old runtimes are not affected by this change.